apps/coscrad-frontend/src/components/resources/media-items/media-item-index.presenter.tsx

```diff
@@ -0,0 +1,11 @@
+import { MediaItemIndexState } from '../../../store/slices/resources/media-items/types';
+import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
+
+export const MediaItemIndexPresenter = (mediaIndexResult: MediaItemIndexState) => {
+    /**
+     *  TODO [https://www.pivotaltracker.com/story/show/183681839]
```

Comment:
Maybe when working on the commands and dynamic form generation is the time to do this.